cmd/ocsp-responder/main.go

```diff
@@ -166,6 +179,9 @@ func (src *dbSource) Response(req *ocsp.Request) ([]byte, http.Header, error) {
 		src.log.AuditErrf("Looking up OCSP response: %s", err)
 		return nil, nil, err
 	}
+	if !src.filter.checkResponse(req, certStatus) {
+		return nil, nil, bocsp.ErrNotFound
```

Comment:
Let's log a Warn in this case. It's user-triggerable, but it should be rare enough in practice that it suggests something is up (perhaps a bug on our end) and we should investigate.

Reply:
Done. Also updated one debug log below to warning.